libevm/VM.cpp

```diff
@@ -621,6 +621,66 @@ void VM::interpretCases()
 		}
 		NEXT
 
+#ifdef EVM_USE_BITSHIFT
+		CASE(SHL)
+		{
+			ON_OP();
+			updateIOGas();
+
+			if (m_SP[0] >= 256)
+				m_SPP[0] = 0;
+			else
+				m_SPP[0] = m_SP[1] << uint64_t(m_SP[0]);
```

Comment:
Could also just use `unsigned` and not the specific `uint64_t` since it always fits into 8 bits.